src/features/offerv2/components/OfferContent/OfferContent.tsx

```diff
@@ -147,6 +154,13 @@ export const OfferContent: FunctionComponent<Props> = ({ offer, searchGroupList,
         <OfferPlace offer={offer} isEvent={subcategory.isEvent} />
         <Spacer.Column numberOfSpaces={8} />
 
+        {isOfferAMovieScreening ? (
```

Comment:
tu peux mettre le feature flag autour de tout l'objet non ?

Reply:
Oui tu peux tout passer à l'intérieur avec la condition et surtout le Spacer sinon ca va ajouter un espace vide quand le FF sera désactivé

Reply:
@TristanMaurier98 @tanguyprache si je passe le FF autour de la condition, j'ai l'erreur TS suivant `Type 'Element | null' is not assignable to type 'Element'.` et ça ne change rien si j'utilise un && au lieu de la ternaire.

Comment vous fixez ce genre d'erreurs d'habitude ?

<img width="691" alt="Capture d’écran 2024-03-04 à 13 13 16" src="https://github.com/pass-culture/pass-culture-app-native/assets/158567429/dd14d263-522b-4696-843d-77f73d7d0468">

Reply:
Alors que là je n'ai pas l'erreur 

<img width="768" alt="Capture d’écran 2024-03-04 à 13 17 43" src="https://github.com/pass-culture/pass-culture-app-native/assets/158567429/0dbdfa8b-a750-4d69-b915-fcd681816cb9">